gatsby-node.ts

```diff
@@ -18,3 +19,11 @@ export const createSchemaCustomization: GatsbyNode['createSchemaCustomization']
   `;
   createTypes(typeDefs);
 };
+
+// Gatsby doesn't allow you to change sitemap filename, so we have to do it manually.
+// https://github.com/gatsbyjs/gatsby/issues/31515
+exports.onPostBuild = () => {
+  if (fs.existsSync('./public/docs/sitemap-index.xml')) {
+    fs.renameSync('./public/docs/sitemap-index.xml', './public/docs/sitemap.xml');
```

Comment:
Hey @snikidev just FYI, but this is not necessary. Docs are served through the website Rails app, and we can map routes there (we already do for Voltaire sitemap, for example https://github.com/ably/website/blob/main/config/routes.rb#L665). 

In fact, this sitemap will not be accessible on ably.com without routing additions on the website.

Reply:
I kind of went through the same thought process but I actually quite like this as it is. If someone comes into the docs area searching for `sitemap.xml` they will find this line and understand exactly how this is setup. But yes, you are right we will need the Ruby route. Shall I create a ticket for it?

Reply:
Yup, a ticket would be good, otherwise sitemap won't be accessible at all :) I'll leave you guys to decide what you want to do. For Voltaire we even skip the index https://github.com/ably/website/blob/main/app/services/ably/backends/cms.rb#L165-L171